libs/libcommon/include/common/StringRef.h

```diff
@@ -10,11 +10,11 @@
 
 #include <city.h>
 
-#if __SSE2__
+#if defined(__SSE2__)
```

Comment:
This is true with:
```#define __SSE2__ 0```

Reply:
This is Ok, because __SSE2__ is either undefined or defined to 1.